.github/workflows/protoc_plugin.yaml

```diff
@@ -0,0 +1,50 @@
+name: protoc_plugin
+permissions: read-all
+
+on:
+  pull_request:
+    branches:
+    paths:
+      - '.github/workflows/protoc_plugin.yaml'
+      - 'protoc_plugin/**'
+      - 'tool/setup.sh'
+  push:
+    branches: [ master ]
+    paths:
+      - '.github/workflows/protoc_plugin.yaml'
+      - 'protoc_plugin/**'
+      - 'tool/setup.sh'
+  schedule:
+    - cron: '0 0 * * 0' # weekly
```

Comment:
Same as above, not sure why we want periodic jobs.

Reply:
See above.